src/components/pages/Discover.tsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * The discover tab is where members can learn more about
+ * systemic inequality and how solutions like UBI can
+ * help prevent it! More information about Raha and
+ * partnerships can be pushed here.
+ */
+import * as React from "react";
+import {
+  View,
+  Text,
+  TouchableHighlight,
+  WebView as WebViewNative,
+  TextProps,
+  ScrollView,
+  Linking
+} from "react-native";
+
+import { MemberSearchBar } from "../shared/MemberSearchBar";
+import { NavigationScreenProp, withNavigation } from "react-navigation";
+import { RouteName } from "../shared/Navigation";
+import { SafeAreaView } from "../../shared/SafeAreaView";
+
+export const DiscoverWebView: React.StatelessComponent = ({
+  navigation
+}: any) => {
+  return <WebViewNative source={{ uri: navigation.getParam("uri") }} />;
+};
+
+type DiscoverCardRaw = {
+  header?: string | string[];
+  title: string | string[];
+  footer?: string | string[];
+  uri: string;
+};
+
+type DiscoverCard = {
+  header?: string;
+  title: string;
+  footer?: string;
+  uri: (navigation: NavigationScreenProp<{}>) => void;
+};
+
+function convertUriToCallback(uri: string) {
+  if (uri.startsWith("https:")) {
+    return (navigation: NavigationScreenProp<{}>) => {
+      navigation.navigate(RouteName.DiscoverWebView, { uri });
+    };
+  }
+  if (uri.startsWith("mailto:")) {
+    return (navigation: NavigationScreenProp<{}>) => {
+      Linking.openURL(uri); // TODO check canOpen
+    };
+  }
+  throw Error(`Invalid uri ${uri}, unsupported protocol`);
```

Comment:
If we're throwing an error we probably should be careful about not crashing the app by calling this - so try catch around whatever calls this, for now it can just console.error and then return some empty value